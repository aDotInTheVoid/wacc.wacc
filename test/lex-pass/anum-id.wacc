int fn2(int r0, int r1) is
    int a = 0
done