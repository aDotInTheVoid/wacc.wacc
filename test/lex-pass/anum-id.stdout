3 int int
3 identifier fn2
1 lparen (
3 int int
2 identifier r0
1 comma ,
3 int int
2 identifier r1
1 rparen )
2 is is
3 int int
1 identifier a
1 assign =
1 number 0
4 done done
0 eof 
